pep8; removed unused vars

aledista opened the pull request.

kastiglione commented:
Thanks for the pull request! Was this done using a tool? Could you update the pull request to leave out all the indentation, whitespace, and line wrapping changes? It would be great to remove unused vars, but I'd rather avoid reformatting the entire code base.

aledista commented:
I changed my commit. I left out style conventions and removed unused imports, vars and fixed a couple of minor issues. I generally use PyCharm to identify issues like that.

kastiglione commented on commands/FBFlickerCommands.py:
```diff
@@ -108,11 +106,6 @@ def inputCallback(self, input):
-      isMac = runtimeHelpers.isMacintoshArch()
-
-      if isMac:
-        recursionName = '_subtreeDescription'
```
This block shouldn't be removed. `recursionName` is used below.

kastiglione commented on fblldb.py:
```diff
@@ -129,13 +127,12 @@ def helpForCommand(command, filename):
         optionFlag = option.longName
       else:
-        optionFlag = optiob.shortName
+        optionFlag = option.shortName
```
:joy:

aledista commented:
Readded ;)

kastiglione commented:
Thanks again for the pull request!

aledista commented:
You're welcome!
